reference_interfaces/CMakeLists.txt

```diff
@@ -33,14 +33,31 @@ rosidl_generate_interfaces(${PROJECT_NAME}
 
 ament_auto_package()
 
+# remove the below lines if fix to #143 is backported to older ROS distros
 # fix rosidl_generator_py bug #143
 # https://github.com/ros2/rosidl_python/issues/143
 set(GENERATED_FILE "${CMAKE_CURRENT_BINARY_DIR}/rosidl_generator_py/${PROJECT_NAME}/msg/_transmission_stats.py")
 
-message(STATUS "checking generated file: ${GENERATED_FILE}")
-add_custom_command(
-  TARGET ${PROJECT_NAME}__python
-  POST_BUILD
-  COMMAND sed -i "s/all(val >= 0 and val) < 256/all(ord(val) >= 0 and ord(val) < 256/" ${GENERATED_FILE}
-  COMMENT "Check generated IDL files for extra parenthesis..."
-  VERBATIM)
+set(DISTROS_WITH_FIX
+  "humble")
```

Comment:
@christophebedard this can be used in the future once the fix is actually released to `rolling` / other distros. We can then remove the logic below with the renamed target.